### PR TITLE
py-boltons: new port

### PR DESCRIPTION
**Pull request by harens**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.15.6 19G73
xcode-select version 2373

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by herbygillot** [comment]
Being that this is a Python _library_, and not command-line utility, this port should be `py-` -prefixed, and `depends_build` moved into the `if {${name} ne ${subport}}` block as is done in the style of other `py-` -prefixed ports.

**Comment by harens** [comment]
Thanks for your suggestions @herbygillot. I have taken them into account and am just waiting for the tests to hopefully pass.

File: python/py-boltons/Portfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+description         Constructs, recipes, and snippets which extend the Python standard library
+long_description    ${description}
+
+homepage            https://boltons.readthedocs.org/
+
+checksums           rmd160  26dbb3437f581226f1b3772b2c0d5bfddec4de04 \
+                    sha256  dd362291a460cc1e0c2e91cc6a60da3036ced77099b623112e8f833e6734bdc5 \
+                    size    172910
+
+python.versions     27 35 36 37 38
```
Comment (mf2k) [inline python/py-boltons/Portfile:25]: There is no need for the older pythons. Just use version 38 to start and we will add new python versions as they become available.

**Review by mf2k — CHANGES_REQUESTED** [review]

File: python/py-boltons/Portfile (hunk 1 above)
Comment (harens) [reply python/py-boltons/Portfile]: Sure thing, thanks.

**Review by mf2k — APPROVED** [review]

**Comment by ryandesign** [comment]
@herbygillot Next time you squash and merge please edit the commit message to reflect only the end result of all the changes, not listing individual fixes that were made in the PR that are not visible in the final squashed commit.

**Comment by herbygillot** [comment]
> @herbygillot Next time you squash and merge please edit the commit message to reflect only the end result of all the changes, not listing individual fixes that were made in the PR that are not visible in the final squashed commit.

Good catch @ryandesign, noted.

**Pull request merged**